src/rockstor/storageadmin/views/rockon_helpers.py

```diff
@@ -292,7 +305,8 @@ def owncloud_install(rockon):
                         '-e', 'HTTPS_ENABLED=true'])
             cmd.extend(['-e', 'DB_USER=%s' % db_user, '-e',
                         'DB_PASS=%s' % db_pw, ])
-        cmd.append(c.dimage.name)
+        image_name_plus_tag = c.dimage.name + ':' + c.dimage.tag
```

Comment:
My only concern is that this function does not seem to be in use anymore. It doesn't hurt to also implement your changes there, of course.

Reply:
See my previous comment. Pretty sure they are used but would be great if you confirmed this as well.